src/server.ts

```diff
@@ -320,11 +319,10 @@ app.get(
 app.get(
   '/_/done',
   wrap(async (req, res) => {
-    // eslint-disable-next-line node/no-deprecated-api
-    const parsed = url.parse(req.url, true);
-    const query = parsed.query || {};
-
-    const handoff = await datastore.getHandoffKey(query.ott + '');
+    const parsed = new URL(req.url);
```

Comment:
change
`new URL(req.url);`
to 
`new URL(req.url,'http://a.a');`
to prevent url parse error